positron/comms/data_explorer-backend-openrpc.json

```diff
@@ -1003,63 +1036,88 @@
 					}
 				}
 			},
-			"column_histogram": {
+			"column_histogram_params": {
 				"type": "object",
-				"description": "Result from a histogram profile request",
+				"description": "Parameters for a column histogram profile request",
 				"required": [
-					"bin_sizes",
-					"bin_width"
+					"num_bins"
 				],
 				"properties": {
-					"bin_sizes": {
+					"num_bins": {
+						"type": "integer",
+						"description": "Number of bins in the computed histogram"
+					},
+					"quantiles": {
```

Comment:
Should the computed `quantiles` be part of the output? Currently they seem to be unused.